meetings/working-groups/extensions/anonymous-extension-declarations.md

```diff
@@ -0,0 +1,368 @@
+
+}
+```
+
+## Motivation
+
+This design starts from a set of assumptions about what matters more or less to users of the feature, deriving a design that meets them. Those assumptions, though gathered from the criticisms of both "type-based" and "member-based" proposals, are certainly debatable and won't all be agreed upon. They represent the beliefs and opinions of the author. Let's go through them below, along with a sneak peek at how they drive the design.
+
+### Type names and grouping
+
+* Developers want to be able to group a collection of related extension members under just one type name, even if they span multiple different underlying types. People would resent having to come up with multiple separate type names for different underlying types, and having to navigate those many names for disambiguation, `using static` inclusion etc.
+* Separate type names for different receiver types won't be necessary for disambiguation - they aren't today.
+
+This implies an overarching grouping entity that carries no inherent information beyond simply a type name. The proposed design reuses top-level non-generic static classes for this purpose.
+
+### Relationship to classic extension methods
+
+* People would like to be able to place their new extension members together with their existing extension methods.
+* People do appreciate being able to declare non-extension static members next to extension declarations.
+* People are better off leaving their classic extension methods as static method declarations so that the static invocation signature is obvious, and to avoid churn and risk.
+* People will, however, expect lookup to work very closely similar between old and new extension methods.
+
+Because the proposed design keeps using non-generic static classes as the overarching grouping entity, new extension declarations can sit side by side with old extension methods and non-extension static members. However, new extension methods do not work exactly the same as old ones. (Although do see the "Alternatives" section).
+
+### Type parameters and underlying types
+
+* Underlying types belong together with their type parameter declarations, and it would be confusing to separate them.
+* People will resent the verbosity of having to repeat underlying types and accompanying type parameters for each member.
+* Sometimes there's a need to specify different parameter details (ref-ness, nullability, ...) for the same underlying type.
+
+The design uses anonymous extension declarations within static classes. These anonymous extension declarations specify type parameters, underlying type and any parameter details for the extension member declarations they contain.
+
+### Member declarations
+
+* Extension member declarations should look identical to corresponding member declarations in classes and structs.
+* Parameter names for underlying values aren't important and people will resent the forced verbosity of having to specify them.
```

Comment:
i somewhat disagree with this.  I think there are places for both.  I do agree that some will resent being forced to name.  But i think for others taht will be a virtue.  It seems like supporting both seems like a possible future (which i do see you mention below).